digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -1,16 +1,20 @@
 package fi.liikennevirasto.digiroad2
 
+import java.time.format.DateTimeFormatter
```

Comment:
I will remove "java.time.format.DateTimeFormatter" and the "fi.liikennevirasto.digiroad2.masstransitstop.oracle.{Queries, Sequences}" because they are not being used. We are already using the "org.joda.time.format._" lib for dateFormat. 
Thank you